### DISCUSSION TITLE
Debugging TrustWallet browser for Android

### DISCUSSION THREAD

**Discussion by TheJuze** [trustwallet/wallet-core#3788] [Q&A]
Hi! I'm developing frontend and trying to debug TrustWallet mobile browser crash  as on others mobile browsers, but i can't see any action in "chrome://inspect" tab. Any other way to see console.logs?

**Comment by satoshiotomakan** [comment]
This issue is not related to the wallet core repository, please create a ticket here if u believe there is a bug in the application: https://support.trustwallet.com/en/support/tickets/new
